README.md

```diff
@@ -149,11 +152,9 @@ trait MyErrorHandler extends OAuthErrorHandler {
 val tokens = new OAuth2Endpoint with MyErrorHandler
 ```
 
-By default errors are serialized in `WWW-Authenticate` header while access tokens are serialized as simple string in 
-the response body.
+By default errors are serialized in `WWW-Authenticate` header while access tokens are serialized as
+simple string in the response body.
 
-There are also two predefined traits: `OAuthErrorInJson` and `OAuthTokenInJson` which serializes everything into JSON objects.
-
-----
-By Vladimir Kostyukov, http://vkostyukov.ru
+There are also two predefined traits: `OAuthErrorInJson` and `OAuthTokenInJson` which serializes
+everything into JSON objects.
```

Comment:
There are also two predefined trait which serialize everything in JSON objects (i.e. ```OAuthErrorInJson``` and ```OauthTokenInJosn```).